products/sle/main.pm

```diff
@@ -646,6 +646,7 @@ sub load_virt_feature_tests {
     }
 }
 
+set_var('VERSION', '16.0') if (check_var('VERSION', '15.99'));
```

Comment:
@tinawang123 can you file a subticket to remove this workaround once production build 16.0 comes out?

Reply:
Thanks, @tinawang123 would you please add some description here why you do this change?